pipelines/toast_ground_sim.py

```diff
@@ -1310,6 +1339,94 @@ def main():
 
     localpix, localsm, subnpix = get_submaps(args, comm, data)
 
+    local_dets = extract_local_dets(data)
+    print(local_dets)
+
+    run_pysm = True
+    if run_pysm:
+
+        # Simulate input sky
+
+        dist_rings = tm.DistRings(comm.comm_rank,
+                            nside = args.nside,
+                            nnz = 3)
+
+        pysm_sky_components = [
+            'synchrotron',
+            'dust',
+            'freefree',
+            'cmb',
+            'ame',
+        ]
+        pysm_sky_config = dict()
+        for component_model in args.input_pysm_model.split(','):
+            full_component_name = \
+                    [each for each in pysm_sky_components
+                     if each.startswith(component_model[0])][0]
+            pysm_sky_config[full_component_name] = component_model
+
+        bandpasses = {}
+        fwhm_deg = {}
+        N_POINTS_BANDPASS = 10  # possibly take as parameter
+        for det in local_dets:
+            bandcenter, bandwidth, fwhm_deg[det] = extract_detector_parameters(det, schedules)
+            bandpasses[det] = \
+                (np.linspace(bandcenter-bandwidth/2,
+                             bandcenter+bandwidth/2,
+                             N_POINTS_BANDPASS),
+                 np.ones(N_POINTS_BANDPASS))
+        pysm_sky = tm.PySMSky(local_pixels=dist_rings.local_pixels,
+                              nside=args.nside,
+                              pysm_sky_config=pysm_sky_config,
+                              bandpasses=bandpasses)
+        local_maps = dict()  # FIXME use Cache instead
+        pysm_sky.exec(local_maps, out="sky")
```

Comment:
This is not a toast operator, so is not writing timestreams to the toast.tod.cache() for every observation.  So what is the "out" parameter used for?